File: Selenium/Activities/Activity5.py
from selenium import webdriver
from selenium.webdriver import ActionChains
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService;
from webdriver_manager.firefox import GeckoDriverManager

service =FirefoxService(GeckoDriverManager().install())
with webdriver.Firefox(service = service) as driver:
    driver.get("https://v1.training-support.net/selenium/input-events")
    print("home Page title is :",driver.title)
    action = ActionChains(driver)
    action.click().pause(2).perform()
    left_click_Text  = driver.find_element(By.CLASS_NAME,'active').text
    action.double_click().pause(2).perform()
    double_click_Text  = driver.find_element(By.CLASS_NAME,'active').text
    action.context_click().pause(2).perform()
    right_click_Text  = driver.find_element(By.CLASS_NAME,'active').text
    print("Left click is :" + left_click_Text);
    print("Double click is :" + right_click_Text);
    print("Right click is :" + double_click_Text);
    driver.close()

File: Selenium/Activities/Activity5.java
package activities;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.By;
import org.openqa.selenium.firefox.FirefoxDriver;
import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.interactions.Actions;

public class Activity5 {
    public static void main(String[] args){
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();
        driver.get("https://v1.training-support.net/selenium/input-events");
        System.out.println("Title of the Page :" + driver.getTitle());
        Actions action = new Actions(driver);
        action.click().pause(1000).build().perform();
        String front_text = driver.findElement(By.className("active")).getText();
        action.doubleClick().pause(1000).build().perform();
        String randomSide_text = driver.findElement(By.className("active")).getText();
        action.contextClick().pause(1000).build().perform();
        String cubeFront_text = driver.findElement(By.className("active")).getText();
        System.out.println("Left click is :" + front_text);
        System.out.println("Double click is :" + randomSide_text);
        System.out.println("Right click is :" + cubeFront_text);
        driver.close();


    }
}
